v20.1/admin-ui-storage-dashboard.md

```diff
@@ -1,35 +1,57 @@
 toc: true
 ---
 
-The **Storage** dashboard in the CockroachDB Admin UI lets you monitor the storage utilization for your cluster. To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** on the left-hand navigation bar, and then select **Dashboard** > **Storage**.
+The **Storage** dashboard lets you monitor the storage utilization of your cluster. 
+
+To view this dashboard, [access the Admin UI](admin-ui-access-and-navigate.html#access-the-admin-ui), click **Metrics** in the left-hand navigation, and select **Dashboard** > **Storage**.
 
 {% include {{ page.version.version }}/admin-ui/admin-ui-metrics-navigation.md %}
 
 The **Storage** dashboard displays the following time series graphs:
 
 ## Capacity
 
+You can monitor the **Capacity** graph to determine when additional storage is needed (e.g., by scaling your cluster). 
+
 <img src="{{ 'images/v20.1/admin_ui_capacity.png' | relative_url }}" alt="CockroachDB Admin UI Capacity graph" style="border:1px solid #eee;max-width:100%" />
 
-You can monitor the **Capacity** graph to determine when additional storage is needed.
+Metric | Description
+--------|--------
+**Capacity** | The maximum store size. This value may be set per node using [`--store`](cockroach-start.html#store). If a store size has not been set, this metric displays the actual disk capacity. See [Capacity metrics](#capacity-metrics).
+**Available** | The free disk space available to CockroachDB data.
+**Used** | The disk space in use by CockroachDB data. This excludes the Cockroach binary, operating system, and other system files.
+
+### Capacity metrics
+
+The **Capacity** graph displays disk usage by CockroachDB data in relation to the maximum [store](architecture/storage-layer.html) size, which is determined as follows:
+
+- If a store size was specified using the [`--store`](cockroach-start.html#store) flag when starting nodes, this value is used as the limit for CockroachDB data.
+- If no store size has been explicitly set, the actual disk capacity is used as the limit for CockroachDB data.
 
-- In the node view, the graph shows the maximum allocated capacity, available storage capacity, and capacity used by CockroachDB for the selected node.
+The **available** capacity thus equals the amount of empty disk space, up to the value of the maximum store size. The **used** capacity refers only to disk space occupied by CockroachDB data, which resides in the store directory on each node.
 
-- In the cluster view, the graph shows the maximum allocated capacity, available storage capacity, and capacity used by CockroachDB across all nodes in the cluster.
+The disk usage of the Cockroach binary, operating system, and other system files is not shown on the **Capacity** graph.
 
-On hovering over the graph, the values for the following metrics are displayed:
+{{site.data.alerts.callout_danger}}
```

Comment:
This is a red "warning" but some other uses of this include are a blue "info".  Intentional?  (Could see why since this is the storage page)